tests/checks/integration/test_fluentd.py

```diff
@@ -79,3 +79,23 @@ def test_fluentd_with_tag_by_plugin_id(self):
         self.assertServiceCheckOK(
             self.check.SERVICE_CHECK_NAME, tags=['fluentd_host:localhost', 'fluentd_port:24220'])
         self.coverage_report()
+
+    def test_fluentd_with_custom_tags(self):
+        config = {
+            "init_config": {
+            },
+            "instances": [
+                {
+                    "monitor_agent_url": "http://localhost:24220/api/plugins.json",
+                    "tag_by": "plugin_id",
+                    "tags": ["tag1", "tag2"]
+                }
+            ]
+        }
+        self.run_check(config)
+        for m in self.CHECK_GAUGES:
+            self.assertMetric('{0}.{1}'.format(self.CHECK_NAME, m), tags=['tag1', 'tag2', 'plugin_id:plg1'])
+            self.assertMetric('{0}.{1}'.format(self.CHECK_NAME, m), tags=['tag1', 'tag2', 'plugin_id:plg2'])
```

Comment:
As per the comment I made above, I think these might fail due to how you're appending the plugin tags to `custom_tags`.